test/e2e/node/kubelet.go

```diff
@@ -310,9 +310,7 @@ var _ = SIGDescribe("kubelet", func() {
 		})
 
 		for _, itArg := range deleteTests {
-			name := fmt.Sprintf(
-				"kubelet should be able to delete %d pods per node in %v.", itArg.podsPerNode, itArg.timeout)
-			ginkgo.It(name, func() {
+			ginkgo.It("should let Kubelet delete x pods per node in n minutes", func() {
```

Comment:
This won't work, because now we'll end up with `len(deleteTests)` identically named test cases.  The thing to do would be unroll the loop.

Which should be pretty easy in this case, because `deleteTests` appears to be a single item array.  It has been this way since the test was moved to this file in 2017-07-25, so it doesn't seem like we need to keep the struct around

Reply:
I've just removed `deleteTests` now from this test in 03cb4bf26f6. No more unnecessary loop :smile: